docs/source/reference/changelog.md

```diff
@@ -20,6 +20,76 @@ Contributors to major version bumps in JupyterHub include:
 
 ## [Unreleased]
 
+## 5.3
+
+### 5.3.0 - 2025-04
+
+([full changelog](https://github.com/jupyterhub/jupyterhub/compare/5.2.1...HEAD))
```

Comment:
Besides updating these lines, :+1: on proceeding with a pre-release!

Reply:
yup, these should be updated when making the final release.